src/sentry/sentry_metrics/use_case_id_registry.py

```diff
@@ -63,4 +61,4 @@ def extract_use_case_id(mri: str) -> UseCaseID:
         use_case_str = matched.group(2)
         if use_case_str in {id.value for id in UseCaseID}:
             return UseCaseID(use_case_str)
-    raise ValidationError(f"Invalid mri: {mri}")
+    raise ValueError(f"Invalid mri: {mri}")
```

Comment:
A separate, general fix that I noticed should be made. I think we only want to raise a ValidationError when explicit JSON schema validation fails